dockerfiles/init/modules/openshift/files/scripts/che-openshift.yml

```diff
@@ -95,35 +95,19 @@ items:
     CHE_HOST: ${DEFAULT_OPENSHIFT_NAMESPACE_URL}
     CHE_WEBSOCKET_ENDPOINT:  ws://${DEFAULT_OPENSHIFT_NAMESPACE_URL}/api/websocket
     CHE_PORT: "8080"
-    DOCKER_HOST: tcp://172.18.0.1:2375
-    hostname-http: ${PROJECT_NAME}-che.${DOMAIN}
-    workspace-storage: /home/user/che/workspaces
-    workspace-storage-create-folders: "false"
     local-conf-dir: /etc/conf
-    openshift-serviceaccountname: che
-    che-server-evaluation-strategy: docker-local-custom
     che.logs.dir: /data/logs
-    che.docker.server_evaluation_strategy.custom.template: <serverName>-<if(isDevMachine)><workspaceIdWithoutPrefix><else><machineName><endif>-<externalAddress>
-    che.docker.server_evaluation_strategy.custom.external.protocol: https
     che.predefined.stacks.reload_on_start: "true"
     log-level: INFO
-    docker-connector: openshift
     port: "8080"
     remote-debugging-enabled: "false"
     che-oauth-github-forceactivation: "true"
-    workspaces-memory-limit: 2300Mi
-    workspaces-memory-request: 1500Mi
     enable-workspaces-autostart: "false"
     keycloak-oso-endpoint: ${KEYCLOAK_OSO_ENDPOINT}
```

Comment:
Would be awesome to have all these values in the same format, e.g. KEYCLOAK_GITHUB_ENDPOINT

Reply:
I'll rework it